### PR TITLE
Add a /health endpoint

### PR DESCRIPTION
**Pull request by uzi**
👓 @schwers

### CODE REVIEW

**Comment by uzi** [comment]
💇

File: src/server/meta/index.js [hunk 1]
```diff
@@ -54,6 +54,10 @@ export default (router, apiOptions) => {
     ctx.type = 'application/json';
   });
 
+  router.get('/health', (ctx) => {
```
Comment (schwers) [inline src/server/meta/index.js]: 🔕 I'd say "you don't need parens here" but it looks like the rest of the file has it, good cleanup to do later

**Comment by schwers** [comment]
:fish:

File: src/server/meta/index.js (hunk 1 above)
Comment (uzi) [inline src/server/meta/index.js]: 👍

**Pull request merged**